test.js

```diff
@@ -0,0 +1,58 @@
+const assert = require('assert');
+Object.freeze(assert);
+
+const sumOfOther = require('./src/sumOfOther'),
+      make = require('./src/make'),
+      recursion = require('./src/recursion');
+
+describe('rsschool_codejam_3 tests', function () {
```

Comment:
It wasn't in requirements but it's good when test code have the same style as the main code (at least 2 and 4 spaces)